Android & PC some fixes

elifian opened the pull request:
1. Fixed the colors of the avatars.  
<img src="https://github.com/user-attachments/assets/7e5347d1-7c76-4f6f-aea6-919f768dc0e0" alt="image" width="200" height="300">

2. Background of the player.  
<img src="https://github.com/user-attachments/assets/b78e66b1-0b03-43e4-aaa9-0ebd5e2c8b0a" alt="image" width="200" height="150">

3. The color of the emoji status has been changed to white.
<img src="https://github.com/user-attachments/assets/256615c1-3208-4ce4-a3a1-93d53d9ba0fb" alt="image" width="200" height="50">
<br><br>
<img src="https://github.com/user-attachments/assets/17fdc086-c419-464b-9fb5-bd15c0900be1" alt="image" width="200" height="100">

4. Fixed the send video button (it was completely white).
<img src="https://github.com/user-attachments/assets/44121e69-72cd-40c6-bc7a-a3047239a90b" alt="image" width="200" height="250">

elifian commented:
This is for users who will be passing by :)
https://t.me/addtheme/dark_shell_2

maximilionus commented:
You did a great job, thank you for fixing this stuff. I don't really have much time for this project lately, so it's been in idle state for a while.

Give me a few moments to check the changes.

elifian commented:
I've made some more minor changes to the theme on the PC, on android. You can see it here https://t.me/addtheme/dark_shell_2

Including the white search bar in emoji and in the attach audio menu (android)

elifian commented:
I would also like to thank you for such a perfect theme. Even though it hasn't received updates for a long time. <3

maximilionus commented:
> 4. Fixed the send video button (it was completely white).
> 
> <img alt="image" width="200" height="250" src="https://private-user-images.githubusercontent.com/69606089/399959394-44121e69-72cd-40c6-bc7a-a3047239a90b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYwMDU3MjIsIm5iZiI6MTczNjAwNTQyMiwicGF0aCI6Ii82OTYwNjA4OS8zOTk5NTkzOTQtNDQxMjFlNjktNzJjZC00MGM2LWJjN2EtYTMwNDcyMzlhOTBiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA0VDE1NDM0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc4ZTNiZjIzY2U5OTIxYTA2MDc0ZTJjNzMwN2VlNGEyYjBkMzc4YmQ4YTI5MDRkZGQxZjUzMjVmNmIxNzU4NzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.3lRz8VT9_75Z_zPl0QMZUFWsRXtxdCz_oPI4briY4gk">

Is it possible to make the button's background white and turn the icon itself black, or this will require more complex tweaks?

elifian commented:
> Is it possible to make the button's background white and turn the icon itself black, or this will require more complex tweaks?

Unfortunately, no, I tried. Apparently, the button in the center is static and you can't change its color, so this is the only way to do it...

maximilionus commented:
> Unfortunately, no, I tried. Apparently, the button in the center is static and you can't change its color, so this is the only way to do it...

Oh, okay, that's to be expected.

I see that you also made some changes to the desktop theme. Please add the changes you made to the PR description so I can check them.

And you can also just make new commits without force-pushing - I'm not putting any limits on the commits amount.

elifian commented:
If something is not clear, ask away, my English is not very good 😅.

> And you can also just make new commits without force-pushing - I'm not putting any limits on the commits amount.

I'm just not a fan of creating excessive commits, sorry.

> I see that you also made some changes to the desktop theme. Please add the changes you made to the PR description so I can check them.

## PC:
1. Changed the background color of "Saved Messages" (The same thing is done for the android)
2. The title color has been changed so that it does not stand out from the main color in the theme.
<img src="https://github.com/user-attachments/assets/2597ca86-326b-4466-a1a8-f3608a550035"/>
<br></br>
3. Fixed the colors of the names
<br></br>
<img src="https://github.com/user-attachments/assets/7f934f0c-6632-4888-a173-e0ab90b96521"/>

## Android (Additional changes):
1. Changed the background color of "Saved Messages"
<img src="https://github.com/user-attachments/assets/7c17513e-cf80-4000-b6c0-316240e22a1a" width="400" height="790">
<br></br>
2. The background of the sticker search has been changed
<br></br>
<img src="https://github.com/user-attachments/assets/da2d6831-ca88-4f8a-b592-6a313f430e8d" width="400" height="200">
<br></br>
3. The background of the audio search has been changed
<br></br>
<img src="https://github.com/user-attachments/assets/8906a743-3e96-47fb-a223-3e247fdf38d9" width="400" height="500">
<br></br>
The search bars have the same hex color

elifian commented:
I almost forgot, the color of the "verified account" and "status" has also been changed from blue to white.
![image](https://github.com/user-attachments/assets/93055fa7-37b9-45cc-9f1b-e9fb1599c22d)
![image](https://github.com/user-attachments/assets/1a1d660e-c0b3-4ab0-bc9a-36fee69ab65a)
Both on PC and Android.

maximilionus commented:
So you just went and singlehandedly fixed all the long-standing issues of this theme on multiple platforms. You are a true hero my friend, none the less 👍

maximilionus commented on TDesktop/DarkShell.tdesktop-palette:
Looks amazing, finally the custom colors are here. New title and search bars look very consistent now.

maximilionus commented on TDesktop/DarkShell.tdesktop-theme:
No need to export the theme binary here, I can do it myself later.

maximilionus reviewed (approved):
The desktop fixes are really nice, you did a great job. Already applied the changes to the cloud variant.

The Android variant will take some more time to test out, as it's a bit more tricky to merge correctly.

maximilionus commented:
> If something is not clear, ask away, my English is not very good 😅.

Я отлично понимаю русский. Не стоит утруждаться с переводчиком, можешь дальше на нем писать.

maximilionus commented:
That's it. HUGE **thank you** for such a massive contribution, basically keeping this project alive.

Changes are already applied to the cloud versions of both Android and Desktop themes, repository release will be out in a few days.
